### PR TITLE
feat: copilot frontend e2e

### PR DESCRIPTION
**Pull request by darkskygit**
fix AF-1432

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7694** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7694?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pengx17 — COMMENTED** [review]
how much does it cost when running one run of the ai tests?

**Comment by darkskygit** [comment]
> how much does it cost when running one run of the ai tests?

with the current number of use cases, a complete test make 6 requests, and the required amount is less than $0.01
image

**Comment by darkskygit** [comment]
> how much does it cost when running one run of the ai tests?

with the current number of use cases, a complete test make 19 requests, and the required amount is less than $0.01
image

**Comment by Brooooooklyn** [comment]
@darkskygit I'd like to create a k8s cron job for ai e2e tests (only API integration tests) rather than put it's in CI

**Comment by darkskygit** [comment]
> @darkskygit I'd like to create a k8s cron job for ai e2e tests (only API integration tests) rather than put it's in CI

api test already in other branch, i will make new pr later
and i think ai action's frontend e2e is also necessary, some previous updates of bs have already caused several interruption in the ai action usage process, an e2e action cron can promptly detect these issues.

**Review by Brooooooklyn — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -392,6 +392,49 @@ jobs:
           name: affine
           fail_ci_if_error: false
 
+  copilot-e2e-test:
```
Comment (Brooooooklyn) [inline .github/workflows/build-test.yml:395]: add it to `needs` of `test-done` job

**Review by Brooooooklyn — COMMENTED** [review]
The `Server Copilot E2E Test` takes too much time

**Pull request merged**